### ISSUE TITLE
Box-Manifest 3.5.0 does not support recent BOX 4.4.0

### ISSUE THREAD

**Issue by llaville** [llaville/box-manifest#9]
# Bug report category

- [x] Feature issue.

## Summary
The recent release [BOX 4.4.0](https://github.com/box-project/box/releases/tag/4.4.0) has introduced some internal code changes that break the `box:compile` command.

## Expected behaviour
Be able to compile again your application into a PHAR

## Actual behaviour
Raise a PHP Fatal Error

## Feature issue.

When trying to compile Box-Manifest application itself (version 3.x-dev for Box 4.4.0@b7e493d) with `bin/box-manifest box:compile` command.
We got following PHP error : 

```text
Fatal error: Uncaught Error: Class "KevinGH\Box\PharInfo\PharInfo" not found in /shared/backups/bartlett/box-manifest/src/Console/Command/BoxCompile.php:133
Stack trace:
#0 /shared/backups/bartlett/box-manifest/src/Console/Command/BoxCompile.php(117): Bartlett\BoxManifest\Console\Command\BoxCompile->updateMetadata(Object(Fidry\Console\Input\IO))
#1 /shared/backups/bartlett/box-manifest/vendor/symfony/console/Command/Command.php(326): Bartlett\BoxManifest\Console\Command\BoxCompile->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /shared/backups/bartlett/box-manifest/vendor/symfony/console/Application.php(1063): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /shared/backups/bartlett/box-manifest/vendor/symfony/console/Application.php(320): Symfony\Component\Console\Application->doRunCommand(Object(Bartlett\BoxManifest\Console\Command\BoxCompile), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /shared/backups/bartlett/box-manifest/vendor/symfony/console/Application.php(174): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /shared/backups/bartlett/box-manifest/vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php(54): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /shared/backups/bartlett/box-manifest/vendor/autoload_runtime.php(37): Symfony\Component\Runtime\Runner\Symfony\ConsoleApplicationRunner->run()
#7 /shared/backups/bartlett/box-manifest/bin/box-manifest(36): require_once('/shared/backups...')
#8 {main}
  thrown in /shared/backups/bartlett/box-manifest/src/Console/Command/BoxCompile.php on line 133
```

**Comment by llaville** [comment]
Prepare bug fix release 3.5.1 to limit support to BOX 4.3.x (see commit 56314f6176ce3f2dbca9700e4d9b4b9635c4928b)

**Comment by llaville** [comment]
Latest version (3.5.1) that support BOX v4.3.x is now available !
